src/DssSpell.t.sol

```diff
@@ -974,48 +1049,257 @@ contract DssSpellTest is DssSpellTestBase {
+    GemAbstract        immutable l2susds                = GemAbstract(       0x5875eEE11Cf8398102FdAd704C9E96607675467a);
+    GemAbstract        immutable susd                   = GemAbstract(       addr.addr("SUSDS"));
+    L1TokenBridgeLike  immutable l1bridge               = L1TokenBridgeLike( addr.addr("BASE_TOKEN_BRIDGE"));
+    BaseGovRelayLike   immutable l1govRelay             = BaseGovRelayLike(  addr.addr("BASE_GOV_RELAY"));
+    address            immutable L1_ESCROW              =                    addr.addr("BASE_ESCROW");
+    address            immutable L1_BRIDGE_IMP          =                    addr.addr("BASE_TOKEN_BRIDGE_IMP");
+    address            constant  L2_BRIDGE_IMP          =                    0x289A37BE5D6CCeF7A8f2b90535B3BB6bD3905f72;
+    address            constant  MESSANGER              =                    0x866E82a600A1414e583f7F13623F1aC5d58b0Afa;
+    address            constant  L2_MESSANGER           =                    0x4200000000000000000000000000000000000007;
+
+    address            immutable PIP_ALLOCATOR_SPARK    =                    addr.addr("PIP_ALLOCATOR_SPARK_A");
+    address            immutable ALLOCATOR_ROLES        =                    addr.addr("ALLOCATOR_ROLES");
+    address            immutable ALLOCATOR_REGISTRY     =                    addr.addr("ALLOCATOR_REGISTRY");
+    address            immutable ALLOCATOR_SPARK_BUFFER =                    addr.addr("ALLOCATOR_SPARK_A_BUFFER");
+    address            immutable ALLOCATOR_SPARK_VAULT  =                    addr.addr("ALLOCATOR_SPARK_A_VAULT");
+    address            immutable ALLOCATOR_SPARK_PROXY  =                    addr.addr('SPARK_PROXY');
+    bytes32            constant  ALLOCATOR_ILK          =                    "ALLOCATOR-SPARK-A";
+
+    address            immutable LITE_PSM               =                    addr.addr("MCD_LITE_PSM_USDC_A");
+
+    // ---------- Medians and Validators  ----------
+    address           constant ETH_GLOBAL_VALIDATOR     =                    0xcfC62b2269521e3212Ce1b6670caE6F0e34E8bF3;
+    address           constant MANTLE_VALIDATOR         =                    0xFa6eb665e067759ADdE03a8E6bD259adBd1D70c9;
+    address           constant NETHERMIND_VALIDATOR     =                    0x91242198eD62F9255F2048935D6AFb0C2302D147;
+    address           constant EULER_VALIDATOR          =                    0x1DCB8CcC022938e102814F1A299C7ae48A8BAAf6;
+    address           constant BTC_USD_MEDIAN           =                    0xe0F30cb149fAADC7247E953746Be9BbBB6B5751f;
+    address           constant ETH_USD_MEDIAN           =                    0x64DE91F5A373Cd4c28de3600cB34C7C6cE410C85;
+    address           constant WSTETH_USD_MEDIAN        =                    0x2F73b6567B866302e132273f67661fB89b5a66F2;
+    address           constant MKR_USD_MEDIAN           =                    0xdbBe5e9B1dAa91430cF0772fCEbe53F6c6f137DF;
+
+    function testBaseTokenBridge() public {
```

Comment:
Wouldn't it be better to split this into 2 different parts:
1. General config checks in `t.base.sol`
2. Spell specific tests in `t.sol`
?

Reply:
When we have a standart interface for bridges then we can do it, now we have few different types of bridges